README.md

```diff
@@ -1,3 +1,7 @@
+> **Note**: We’ve migrated a set of Features to the new repo [devcontainers/features](https://github.com/devcontainers/features) and we will now be publishing Images from [devcontainers/images](https://github.com/devcontainers/images).
```

Comment:
This is a great start! A couple of thoughts:

- Maybe we format a bit more boldly so it really catches users eyes? 
- We could give a bit more context on the spec and new repos/org

Example I'm envisioning:

**IMPORTANT NOTE: We're starting to migrate contents of this repo to the [devcontainers org](https://github.com/devcontainers), as part of the work on the [open dev container specification](https://containers.dev).**

**We've currently migrated a set of Features to a new [devcontainers/features](https://github.com/devcontainers/features) repo, and we'll now be publishing images from [devcontainers/images](https://github.com/devcontainers/images).**

**For more details, you can review the [announcement issue](https://github.com/microsoft/vscode-dev-containers/issues/1589).**

Reply:
Nice suggestion. Made changes!